optimum/intel/openvino/configuration.py

```diff
@@ -240,7 +245,7 @@ def __init__(
         sensitivity_metric: Optional[str] = None,
         ignored_scope: Optional[dict] = None,
         num_samples: Optional[int] = None,
-        quant_method: Optional[Union[QuantizationMethod, OVQuantizationMethod]] = OVQuantizationMethod.DEFAULT,
+        quant_method: Union[QuantizationMethod, OVQuantizationMethod] = OVQuantizationMethod.DEFAULT,
```

Comment:
Why not `Optional`?

Reply:
I removed it as currently it's not expected for `quant_method` to be set to None  (`Optional[OVQuantizationMethod]` <-> `Union[OVQuantizationMethod, None]`), let me know if you think this should be a possibility

Reply:
Got it, thanks.